testbed/testbed/load_generator.go

```diff
@@ -134,7 +134,7 @@ func (lg *LoadGenerator) generate() {
 	// Indicate that generation is done at the end
 	defer lg.stopWait.Done()
 
-	if lg.options.DataItemsPerSecond <= 0 {
+	if lg.options.DataItemsPerSecond == 0 {
```

Comment:
why this change? Maybe a separate PR?

Reply:
One of linters or checkers was complaining, don't remember which one.

Reply:
Ah I see DataItemsPerSecond is uint so cannot be negative :)